xml2json/lib.py

```diff
@@ -1,4 +1,7 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
We've [briefly discussed this in the past](https://github.com/dimagi/commcare-hq/commit/57befb3ca35d318df7ac8caab7e2408c4235aeec#commitcomment-15653090), but didn't come to any solid conclusion. Would it be very difficult to prepend every string in this file with `u` instead? (that would probably be the easiest way to defer the discussion without using `unicode_literals` here). Another option is to import `unicode_literals` in every file in this library. I think the hardest, although nicest for long-term is to come to an agreement on whether (and how) we will use `unicode_literals` in all of our code.

Reply:
Is there a fixer that would prepend every string with ```u```?  I didn't see one, so I thought it might be easier to switch everything to use ```unicode_literals```.

One nice part about using ```unicode_literals``` everywhere is that the meaning of the python text won't change when we switch to python 3.

Reply:
I think if we want to use `unicode_literals` in this library then we should add it to all .py files in the repo. If that's not feasible, then don't use it anywhere. Also see my comment on https://github.com/dimagi/commcare-hq/pull/18373#discussion_r148555954